index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <ul class="menu">
+            <li class="twiter-icon">
+                <a href=""><img src="icons/twiter.jpg" alt="twitter"></a>
+            </li>
+            <li class="home-1">
+                <a href=""><img src="icons/home.jpg" alt="home"> </a>
+                <a href="">Home</a>
+            </li>
+            <li class="explore">
```

Comment:
you could put all those repetitive buttons in one single class 🙂 would be a good way to make your code look cleaner